src/apps/lwaftr/lwaftr.lua

```diff
@@ -711,6 +711,7 @@ local function icmpv6_incoming(lwstate, pkt)
       -- Accept all unreachable or parameter problem codes
       local reply = tunnel_unreachable(lwstate, pkt,
                                        constants.icmpv4_host_unreachable)
+      drop(pkt)
       return transmit_icmpv4_reply(lwstate, reply, pkt)
```

Comment:
This is not right.  After drop(pkt) you may not use pkt again, as you do here on line 715.